src/frontend/context/behandlingContext/useBehandlingApi.ts

```diff
@@ -78,6 +78,10 @@ const useBehandlingApi = (
     };
 
     const hentLogg = (): void => {
+        if (behandlingId === undefined) {
```

Comment:
Så du hvorfor denne kan bli undefined? Komponenten bør ikke laste om behandlingId er undefined egentlig. Og for at dette skal funke så må vi også returne så vi ikke prøver å hente allikevel :)

Reply:
La på en else.
Skjønner ikke helt hvorfor det skjer. Vi henter behandlingsid fra urlen så det må være noe krøll der